plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -456,16 +457,31 @@ public IcebergConfig setQueryPartitionFilterRequiredSchemas(Set<String> queryPar
     }
 
     @Min(0)
-    public int getSplitManagerThreads()
+    public int getSplitSourceThreads()
     {
-        return splitManagerThreads;
+        return splitSourceThreads;
     }
 
-    @Config("iceberg.split-manager-threads")
+    @Config("iceberg.split-source-threads")
```

Comment:
use `@LegacyConfig("iceberg.split-manager-threads")` so Trino still starts up with old configuration.

Reply:
I just realized I hadn't pushed the latest version last night, but I switched the split source executor to be a cached threadpool executor instead which removed the need to change the configs or docs.